src/components/ShowMoreScholarshipCard.jsx

```diff
@@ -0,0 +1,38 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+export default function ShowMoreScholarshipCard({ scholarship }) {
+  const getAmount = (amountData) => {
+    if (amountData.min === amountData.max) {
+      return amountData.min;
+    } else {
+      return (
+        '$' + amountData.min.toString() + '-' + '$' + amountData.max.toString()
+      );
+    }
+  };
```

Comment:
FYI, [ScholarshipAmount.toString()](https://github.com/beyondhb1079/s4us/blob/5a111365d8321a3cc2d21d014c160161008a13ba/src/types/ScholarshipAmount.ts#L129-L130) already does this and it's tested. You don't need to reinvent the wheel. https://github.com/beyondhb1079/s4us/blob/5a111365d8321a3cc2d21d014c160161008a13ba/src/types/ScholarshipAmount.ts#L129-L130